index.html

```diff
@@ -0,0 +1,401 @@
+              <p class="blog__excerpt">
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdiet...
+              </p>
+              <a href="#" target="_blank" class="blog__link">READMORE</a>
+            </li>
+            <li class="blog__item">
+              <img src="#" alt="Blog Post 6" class="blog__image" />
+              <span class="blog__tag">NFT</span>
+              <div class="blog__meta">
+                <p class="blog__author">Tony Nguyen</p>
+                <time datetime="2022-06-20" class="blog__date">20 Jun 2022</time>
+              </div>
+              <h3 class="blog__post-title">Sonic live</h3>
+              <p class="blog__excerpt">
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdiet...
+              </p>
+              <a href="#" target="_blank" class="blog__link">READMORE</a>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="about-us">
+        <!--ABOUT US BLOCK-->
+        <h2 class="about-us__title">About Us</h2>
+        <p class="about-us__description">
+          Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
+          dolore...
+        </p>
+        <div class="about-us__list-wrapper">
+          <ol class="about-us__list">
+            <li class="about-us__list-item">
+              <span class="about-us__number">01</span>
+              <h3 class="about-us__list-item-title">High Quality</h3>
+              <p class="about-us__list-item-description">
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry.
+              </p>
+            </li>
+            <li>
+              <span class="about-us__number">02</span>
+              <h3 class="about-us__list-item-title">Big Collection</h3>
+              <p class="about-us__list-item-description">
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry.
+              </p>
+            </li>
+            <li>
+              <span class="about-us__number">03</span>
+              <h3 class="about-us__list-item-title">Top Resource</h3>
+              <p class="about-us__list-item-description">
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry.
+              </p>
+            </li>
+            <li>
+              <span class="about-us__number">04</span>
+              <h3 class="about-us__list-item-title">Big Community</h3>
+              <p class="about-us__list-item-description">
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry.
+              </p>
+            </li>
+          </ol>
+        </div>
+      </section>
+
+      <!--OUR TEAM BLOCK-->
+      <h2 class="team-card__title">Our Team</h2>
+      <section class="team-card">
```

Comment:
`team-card` is name for `article`, not for all `section`.
This is `team-section`, `team-block` or `team-container`, because it's not a card, it's a block.